imap_processing/spice/geometry.py

```diff
@@ -159,3 +161,102 @@ def get_spacecraft_spin_phase(
     # now (dummy implementation)
     # TODO: Replace with actual logic to calculate spin phase
     return np.array(query_met_times, dtype=float)
+
+
+@typing.no_type_check
+@ensure_spice
+def frame_transform(
+    from_frame: SpiceFrame,
+    to_frame: SpiceFrame,
+    et: Union[float, npt.NDArray],
+    position: npt.NDArray,
```

Comment:
I'm not sure if we have any standard way of handling argument order, but this stuck out to me at a quick glance expecting the arrays to be first in the list and the from/to to be last. Not a major preference from me other than to say we should be consistent if we've been doing it this way then we should stick to it.

Reply:
Good point. I was keeping arguments the same as SPICE uses but I'll think a bit about if one way or the other makes more sense.